aten/src/ATen/xpu/detail/XPUHooks.cpp

```diff
@@ -38,7 +44,13 @@ const Generator& XPUHooks::getDefaultXPUGenerator(
 }
 
 Device XPUHooks::getDeviceFromPtr(void* data) const {
+#ifdef _WIN32
+  TORCH_CHECK(
+      false,
+      "Default context is not supported on XPU on Windows. So we can NOT find the ATen device of a pointer.");
```

Comment:
How do we plan to address the default context on Windows? Is it in the next step? If so, suggest to add a TODO here.

Reply:
It will be not supported in a long time. We will take it as a feature enabling in future when it is available in SYCL on Windows.

Reply:
I think it is our next step. @min-jean-cho  Could you help to add `TODO`  on line 47 and line 28, like:
`TODO: We will switch to the default context on Windows to support this feature`